recipes/opus/all/conanfile.py

```diff
@@ -21,14 +22,24 @@ class OpusConan(ConanFile):
         "fPIC": [True, False],
         "fixed_point": [True, False],
         "stack_protector": [True, False],
+        "osce": [True, False],
+        "deep_plc": [True, False],
+        "dred": [True, False],
```

Comment:
```suggestion
```

I would suggest removing these new options. 

When any of them is enabled, it will require DNN support as well:

https://gitlab.xiph.org/xiph/opus/-/compare/v1.4...v1.5.2?from_project_id=36&straight=false#9a2aa4db38d3115ed60da621e012c0efc0172aae_368_401

Which results in the follow error:

```
-- Configuring done (14.7s)
CMake Error at cmake/OpusFunctions.cmake:180 (target_sources):
  Cannot find source file:

    dnn/fargan_data.h

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm
  .ccm .cxxm .c++m .h .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90
  .f95 .f03 .hip .ispc
Call Stack (most recent call first):
  CMakeLists.txt:400 (add_sources_group)
```

The `dnn/fargan_data.h` a generated file, resulting from the pre-training commands using Python, a pre-steps, before building: https://gitlab.xiph.org/xiph/opus/-/blob/v1.5.2/dnn/torch/fargan/README.md#L37 

As those options are OFF by default, even in the project (https://gitlab.xiph.org/xiph/opus/-/blob/v1.5.2/CMakeLists.txt#L86) we don't see it, but it's the scenario that will result in new issues for CCI in the future.